### PR TITLE
Add the possibility to configure the placeholder

### PR DESCRIPTION
**Pull request by ioanlucut**
ref #142: Add the possibility to configure the placeholder only during loading of the image (when the lqip is shown)

Changes related to https://github.com/stereobooster/react-ideal-image/issues/142

### CODE REVIEW

File: src/components/Media/index.js [hunk 1]
```diff
@@ -56,13 +56,19 @@ export default class Media extends PureComponent {
...
+    /** Server Side Rendering mode flag */
+    ssr: PropTypes.bool,
   }
 
   static defaultProps = {
     iconColor: '#fff',
     iconSize: 64,
   }
 
+  get isLoaded() {
```
Comment (stereobooster) [inline src/components/Media/index.js]: I would prefer to use function instead of getter to be consistent

File: src/components/Media/index.js [hunk 2]
```diff
@@ -92,48 +98,47 @@ export default class Media extends PureComponent {
...
+        height={this.props.height}
         ref={ref => (this.dimensionElement = ref)}
       />
     )
   }
 
-  renderNoscript(props) {
-    return props.ssr ? (
+  renderNoscript() {
+    return this.props.ssr ? (
```
Comment (stereobooster) [inline src/components/Media/index.js]: this change is unrelated to the feature. Idea is to move to hooks from classes, so this code will be changed back eventually.

File: src/__tests__/basic.test.tsx [hunk 3]
```diff
@@ -8,7 +8,7 @@ export default class Application extends React.Component<Props, State> {
   render() {
     return (
       <IdealImage
-        theme={{placeholder: {filter: 'blur(3px)'}}}
+        theme={{placeholder: {backgroundRepeat: 'repeat'}, 'placeholder--loading': {filter: 'blur(5px)'}}}
```
Comment (stereobooster) [inline src/__tests__/basic.test.tsx]: shouldn't it be  `no-repeat cover`?

File: src/components/Media/index.js (hunk 2 above)
Comment (ioanlucut) [reply src/components/Media/index.js]: The only reason was to use the `isLoaded` function/getter properly in all other cases, to be consistent.

File: src/components/Media/index.js [hunk 4]
```diff
@@ -148,17 +153,18 @@ export default class Media extends PureComponent {
       <div
         {...compose(
           theme.placeholder,
+          !this.isLoaded && theme['placeholder--loading'],
```
Comment (stereobooster) [inline src/components/Media/index.js]: there are other icons, which will get blurred: Loading | Network error | 404 error | Offline

https://github.com/stereobooster/react-ideal-image/blob/master/introduction.md

File: src/__tests__/basic.test.tsx (hunk 3 above)
Comment (ioanlucut) [reply src/__tests__/basic.test.tsx]: I updated to a more complete example.

File: src/components/Media/index.js (hunk 4 above)
Comment (ioanlucut) [reply src/components/Media/index.js]: True, thanks for pointing it out. Need to think a bit about it.

**Comment by ioanlucut** [comment]
Any idea why Travis is keep failing? Running the tests locally looks green.

**Comment by stereobooster** [comment]
> Any idea why Travis is keep failing? Running the tests locally looks green.

bug in jest. master branch also fails

**Comment by ioanlucut** [comment]
I am not sure if it's a bug in jest or a new feature, as the coverage isn't 100% across the project. Is it ok that I removed the constraints about the testing coverage?

**Comment by ioanlucut** [comment]
> bug in jest. master branch also fails

Why does pass locally, then?

**Comment by stereobooster** [comment]
https://github.com/facebook/jest/issues/5270
